Gemfile

```diff
@@ -99,6 +99,9 @@ gem "secure_headers"
 # DFE formbuilder
 gem "govuk_design_system_formbuilder", ">= 3.0.1"
 
+# DFE ViewComponent library
+gem "govuk-components"
```

Comment:
might be worth pessimistically versioning?!

Reply:
I don't have a strong opinion either way. 

It's semantically versioned, and I'd expect us to be confident we aren't affected by breaking changes in a major upgrade of any gem, but I see no harm in adding it here. 👍